Code/GraphMol/FMCS/testFMCS_Unit.cpp

```diff
@@ -528,6 +529,220 @@ void testTarget_no_10188_49064() {
   BOOST_LOG(rdInfoLog) << "\tdone" << std::endl;
 }
 
+ROMol *embed3DFromSmiles(const char* smiString, const ROMol *scaffoldNoHs,
```

Comment:
Same comments in this file: it would be better to just include the coordinates you want as a mol block here or in a mol/sdf file.

Reply:
Good point. I thought some more about this, and there were two unit tests that were really redundant, so I kept the one that tests a saved SDF.